src/odemis/gui/cont/tabs.py

```diff
@@ -25,15 +25,36 @@
 from __future__ import division
 
 import collections
-from concurrent.futures import CancelledError
-from functools import partial
 import gc
 import logging
 import math
+import os.path
+import time
+from concurrent.futures._base import CancelledError
+from functools import partial
+
 import numpy
+import pkg_resources
+import wx
+# IMPORTANT: wx.html needs to be imported for the HTMLWindow defined in the XRC
+# file to be correctly identified. See: http://trac.wxwidgets.org/ticket/3626
+# This is not related to any particular wxPython version and is most likely permanent.
+import wx.html
+
+import odemis.acq.align
+import odemis.acq.stream as acqstream
+import odemis.gui
+import odemis.gui.cont.acquisition as acqcont
+import odemis.gui.cont.export as exportcont
+import odemis.gui.cont.streams as streamcont
+import odemis.gui.cont.views as viewcont
+import odemis.gui.model as guimod
+import odemis.gui.util as guiutil
+import odemis.gui.util.align as align
 from odemis import dataio, model
 from odemis.acq import calibration, leech
-from odemis.acq.align import AutoFocus, AutoFocusSpectrometer
+from odemis.acq.align import autofocus
```

Comment:
Please just import the autofocus functions from acq.align, and avoid using `autofocus.`